jerry-core/ecma/builtin-objects/ecma-builtin-helpers.h

```diff
@@ -120,6 +120,13 @@ ecma_value_t ecma_date_value_to_time_string (ecma_number_t datetime_number);
 
 #endif /* !CONFIG_DISABLE_DATE_BUILTIN */
 
+#ifndef CONFIG_DISABLE_REGEXP_BUILTIN
+
+ecma_value_t
+ecma_builtin_helper_regexp_read_pattern_str (ecma_value_t pattern_arg, ecma_string_t **pattern_string_p);
```

Comment:
IMHO this fits better into `ecma-regexp-object.h` and `ecma-regexp-object.c`

Reply:
Yes, it would be better. Please modify this, otherwise LGTM.